siriuspy/siriuspy/devices/idff.py

```diff
@@ -663,6 +663,27 @@ def load_config(self, name):
         else:
             raise ValueError('Could not load configuration.')
 
+    def read_setpoints(self, corrdevs=None):
+        """Return corrector SP values."""
+        if corrdevs is None:
+            corrdevs = self._devsch + self._devscv + self._devsqs
+        chs = _IDSearch.conv_idname_2_idff_chnames(self.devname)
+        cvs = _IDSearch.conv_idname_2_idff_cvnames(self.devname)
+        qss = _IDSearch.conv_idname_2_idff_qsnames(self.devname)
+        lcs = _IDSearch.conv_idname_2_idff_lcnames(self.devname)
+        qns = _IDSearch.conv_idname_2_idff_qnnames(self.devname)
+        ccs = _IDSearch.conv_idname_2_idff_ccnames(self.devname)
+        corrs = chs + cvs + qss + lcs + qns + ccs
+        setpoints = dict()
+        for pvname in corrs:
+            # find corrdev corresponding to pvname
+            for dev in corrdevs:
+                if dev.devname in pvname:
+                    spvname = _SiriusPVName(pvname)
+                    # propty = spvname.propty.replace('-SP', '-RB')
+                    setpoints[pvname] = dev[spvname.propty]
```

Comment:
@xresende, I didn't get where in the code `spvname` is associated with the `"Current-SP"` property. Is it missing something here?